plugin/powerControl/powerControl.yaml

```
@@ -0,0 +1,43 @@
+  EnableSleep: true
+  SleepTimeThresholdSeconds: 70     
+  IdleReserveRatio: 0.2               
+  CheckIntervalSeconds: 60   
+  NodeStateCheckIntervalSeconds: 30
+  PowerControlLogFile: "YOUR_LOG_FILE_PATH"
+  NodeStateChangeFile: "YOUR_NODE_STATE_CHANGE_FILE_PATH"
+  ClusterStateFile: "YOUR_CLUSTER_STATE_FILE_PATH"
+
+Predictor:
+  Debug: true
+  URL: "http://localhost:5000"   
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)